.github/workflows/call-backport-label-updater.yaml

```diff
@@ -0,0 +1,24 @@
+---
+    name: Call Backport Label Updater
+    on:
+      pull_request_target:
```

Comment:
I'm not really sure how I would test this, open to ideas. We could always YOLO merge it and then test it out with some real PRs during early v1.15 backporting steps.

Reply:
Switch to `pull_request` in a temporary commit for testing from the PR :)